identity/Cargo.toml

```
@@ -33,11 +33,12 @@ zeroize = { version = "1.6", optional = true }
 ring = { version = "0.16.9", features = [ "alloc", "std"], default-features = false, optional = true }
 
 [features]
-secp256k1 = ["dep:libsecp256k1", "dep:asn1_der", "dep:rand", "dep:sha2", "dep:hkdf", "dep:zeroize"]
-ecdsa = ["dep:p256", "dep:rand", "dep:void", "dep:zeroize", "dep:sec1", "dep:sha2", "dep:hkdf"]
-rsa = ["dep:ring", "dep:asn1_der", "dep:rand", "dep:zeroize"]
-ed25519 = ["dep:ed25519-dalek", "dep:rand", "dep:zeroize", "dep:sha2", "dep:hkdf"]
-peerid = ["dep:multihash", "dep:bs58", "dep:rand", "dep:thiserror", "dep:sha2", "dep:hkdf" ]
+secp256k1 = ["dep:libsecp256k1", "dep:asn1_der", "dep:sha2", "dep:hkdf", "dep:zeroize"]
+ecdsa = ["dep:p256", "dep:void", "dep:zeroize", "dep:sec1", "dep:sha2", "dep:hkdf"]
+rsa = ["dep:ring", "dep:asn1_der", "dep:rand", "dep:zeroize"] # It is ok for RSA to depend on rand as it doesn't compile to WASM and similar environments anyway.
```

```suggestion
rsa = ["dep:ring", "dep:asn1_der", "dep:rand", "dep:zeroize"]
```

It does depend on `rand` though, so I don't think we need this comment :)